influxdb3/client.go

```diff
@@ -187,9 +188,16 @@ func (c *Client) makeAPICall(ctx context.Context, params httpParams) (*http.Resp
 	}
 	err = c.resolveHTTPError(resp)
 	if err != nil {
+		closingErr := resp.Body.Close()
+		if closingErr != nil {
+			slog.Warn(fmt.Sprintf("Failed to close response body on HTTP Error(%s): %s",
+				err.Error(),
+				closingErr.Error()))
+		} else {
+			slog.Debug(fmt.Sprintf("Closed response body on HTTP Error(%s)", err.Error()))
+		}
```

Comment:
Move closing of the response body to the `resolveHTTPError` method and omit useless logging. As it is in the [InfluxDB Go Client v2](https://github.com/influxdata/influxdb-client-go/blob/master/api/http/service.go#L146)

Reply:
Moved, and corresponding tests have been removed.